src/components/dialogs/script-dialog.js

```diff
@@ -129,8 +124,8 @@ const ScriptDialog = ({ id, open, onClose, onError, title, type }) => {
 
     const getCurrentScript = useCallback(
         (currentItemId) => {
-            if (type === elementType.CONTINGENCY_LIST) {
-                getContingencyList(contingencyListSubtype.SCRIPT, currentItemId)
+            if (type === ElementType.CONTINGENCY_LIST) {
+                getContingencyList(ContingencyListType.SCRIPT, currentItemId)
```

Comment:
Call getContingencyList only if currentItemId !== null (like in form-contingency-dialog.js) to avoid a bad request

Reply:
Done